app/Enums/Gender.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+namespace App\Enums;
+
+use BenSampo\Enum\Enum;
+
+/**
+ * @method static static OptionOne()
+ * @method static static OptionTwo()
+ * @method static static OptionThree()
+ */
+final class Gender extends Enum
+{
+    const MALE = 0;
+    const FEMALE = 1;
+    public static function getDescription(mixed $value): string
```

Comment:
```suggestion
    const MALE = 0;
    const FEMALE = 1;

    public static function getDescription(mixed $value): string
```
健太さんのコードは全体的に改行が少ないです。
習慣的な物ですがもうちょっと空けるようにしてください。

Reply:
直接のご指摘、ご説明ありがとうございました。
コントローラー含め1つの処理のまとまりをイメージして改行を設けました。
ご確認よろしくお願い致します。
app/Http/Controllers/UserController.php
app/Http/Controllers/AuthenticationController.php